doc/collate1d.rst

```diff
@@ -112,11 +128,12 @@ followed by a list of spec1d files. An example configuration file is shown below
     Science/spec1d*.fits
     spec1d end
 
-Coadd1D Configuration
---------------------- 
-Coadd1d configuration can be configured in the ``.collate1d`` as shown above, or
-in a separate ``.coadd1d`` file with the same base name as the ``.collate1d`` file.
-:ref:`pypeit_par:Coadd1DPar Keywords`, 
+Coadd1D and Fluxing Configuration
+---------------------------------
+Coadd1d and Fluxing configuration can be configured in the ``.collate1d`` as shown above. 
+Coadd parameters can be specified in a separate ``.coadd1d`` file with the same base name 
+as the ``.collate1d`` file (See :ref:`pypeit_par:Coadd1DPar Keywords`). For the :ref:`pypeit_par:FluxCalibratePar Keywords`
+``pypeit_collate_1d`` will always set ``extrap_sens`` and ``use_archived_sens`` to True.
```

Comment:
These will always be if "flux" is given on the command line or in the config file. use_archived_sens is set because that's the only way pypeit_collate_1d does fluxing, and extrap_sens is set because I kept getting failures because the archived sensfunc didn't exactly cover the wavelengths of the spec1d files.